### ISSUE TITLE
Possible incrrection in DQN & DDQN file

### ISSUE THREAD

**Issue by YwEwanHuang** [keon/deep-q-learning#35]
The DQN algorithm from **NATURE** leverages a target network to update the target Q value for training. 

So I think the code in **ddqn.py** should be code for the DQN algorithm.

**Comment by keon** [comment]
Good catch — confirmed in 6e62819. The previous `ddqn.py` was Nature-2015 DQN-with-target-network (target net for both action selection and value), not Double DQN. The actual DDQN formulation (online net selects, target net evaluates) was even sitting commented out in the file. The active update is now the real van Hasselt 2016 Double DQN.
